### ISSUE TITLE
Tier 8 harvesters bug

### ISSUE THREAD

**Issue by Taccio33** [Dunkmania101/SpatialHarvesters#11]
There seems to be a bug in witch the tier 8 harvesters go faster and faster in harvesting, getting stacks, at a time after 20-30 seconds of running. 
Even weirder if the max stacksize of the inventory is bigger (like with dank nulls) sometimes it gives the max stack size (even the 2B) or just a very big amount. 
This seems to happens only on tier 8s, but maybe it's just a lot slower in tier 7s and down. Tested with the latest version of the mod and the previous one.

**Comment by Dunkmania101** [comment]
Well, I can pretty much confirm that if the speed thing is really happening, it's not exclusive to the tier 8s, since they're all controlled by the same code. They just change some variables to fit themselves, like the speed, outputs, and price. I can't see anything that would cause it though. Did you add an extra space ripper before it started? The outputs are per ripper. As for the stack size issue, my best guess is that it's because of something to do with the inventories since each stack is either of size one or of whatever was dropped by a mob if it's that harvester. That and I'm using a Forge provided method to do the actual insertion of the items. Thanks for your participation, by the way!

**Comment by Dunkmania101** [comment]
I tested the speed thing and didn't see any change, it's pretty fast so I think it was just an illusion. With that and my thinking that there's nothing I can do about the stack sizes, I'll close this for now. Thanks!

**Comment by Taccio33** [comment]
So to answer some questions and add some details: 
- no there was just one space ripper. In my survival I had 4 of them, up to the 7s, then I upgraded and my dank null (maxed out) starter filling in with dimensional shard (rftools) 2B per space at a time. 
After that I testwd in a fresh world with one space ripper in a vanilla chest and it happened 64s per iteration of the machine. 
-resetting the machine (getting a new one for example) seems to change the random output, like one time with coal, one time with gold and so on. 
-this is presenten (at least it seems) in all harvesters that are fast enought, so ore, soil ... And so on. Maybe, because of the fact that they are all the same, I didn't see it for lower tears due to time, I'm probably going to test it overnight with a server to see if it happens with tear 7s. 
-the issue seemed to diminish in the latest update, since before that it was filling chest at a ridiculous speed, where in the latest update is "x stacks" per harvest. Still means that I did get 100k iron in 5 minutes, but ...
If it's not clear I love the mod, and I like how it's a lag free (almost) alternative for most stuff, inspired by woot, envorimental tech  and those sorts of mods, so no problem for the input, I will continue to give because I'd really like to see this mod take off on tech packs now that 1.16 seems to be growing.
